test/service.test.js

```diff
@@ -0,0 +1,69 @@
+// No env set here, uses env set on index.test.js
+
+var service = require('../lib/service.js');
+var dir = __dirname.split('/')[__dirname.split('/').length - 1];
+var file = dir + __filename.replace(__dirname, '');
+var test = require('tape');
+
+// TODO Delete this test / Or rather, switch to using CURRENT_SERVICE
+// TODO Add a test for using CURRENT_SERVICE
+// TODO Ok assumption?????
+// Assumes .env used for testing has all available services configured simultaneously
```

Comment:
@zemccartney what is your plan with these `TODO`s, are you planning to address these in this pr or should we open an issue to work on these in a later pr?

Reply:
heh...sorry for that sloppiness. those are all resolved, I did a pass over the service testing file. I added a handful of new stuff, but all things that make the tests more reliable and readable.